### DISCUSSION TITLE
Art Design Updates

### DISCUSSION THREAD

**Discussion by adamhannigan** [sunflower-land/sunflower-land#1848] [Art]
The art community has come a tremendous way from our initial beta launch in April. Since then we have introduced new style guidelines, dimensions & aesthetics.

Now that we have Sunflower Isles, all SFTs can be placed on a user's land however they want.

Currently, there are a few pieces where the dimensions are too large that we will need to redesign so they fit player's land perfectly.

**SFTs**

When redesigning art, you do not need to follow the exact same style that was used. We are all about iterating and improving on existing designs so please feel free to make some changes 🙏 

1. Observatory - 2x3 grid
2. Sunflower Statue - 2x2 grid
3. Sunflower Tombstone - Colours/aesthetic need to be updated to match current game ✅ @BodenSFLpenguin 
4. Golden Cauliflower - Perspective needs to change so it looks like it can be placed on the ground ✅  @LittleEins 
5. Carrot Sword - 2x2 grid. Needs to be placed on a podium/statue so it can stand up
6. Sunflower Rock - 3x2 grid
7. Nancy - 2x2 grid. Pixels need to be updated to have consistent size as rest of game and dimensions can be wider (similar to other scarecrows)

### Submissions

- Fits game aesthetic and maintains same dimensions
- Original pieces of work
- Pixel sizing is 1x1px
- Dimensions 28px wide and 28px height

_By contributing to the project you agree to the terms and conditions - https://docs.sunflower-land.com/support/terms-of-service. Your art can be used freely by the project and all community members_

When submitting art, please include the following:

Your farm ID (we will make sure the artists get some goodies 🏆 )
A minified version of your art piece
An example of your art on the land

**Rewards**

Unfortunately we cannot airdrop some of these items since they have already been launched. We will ensure the artists do get a care package for their farm with Sunflower goodies ❤️

**Comment by Vergelsxtn** [comment]
Nancy
![nancy1](https://user-images.githubusercontent.com/99605039/206821789-ab7bb40d-3887-4847-82b4-8cfa0c288274.png)
![nancy](https://user-images.githubusercontent.com/99605039/206821790-bf74a987-4263-40fb-8c0a-89ce298891c8.png)

**Reply by adamhannigan** [reply]
Love it!

**Comment by dubuminho** [comment]
**Golden Cauliflower**

![samplegcauli](https://user-images.githubusercontent.com/107245706/206835017-be270b61-56a7-4b1e-be0d-8759cf5ca775.png)


![securedgcauli](https://user-images.githubusercontent.com/107245706/206835113-4f921772-6e37-4a58-8cd0-a1c5b5e33ca4.png)

![gcaulistatue](https://user-images.githubusercontent.com/107245706/206836021-e8c9b007-bc57-428f-b2ef-577f752f28c3.png)

**Sunflower Rock**
![samplesfrock](https://user-images.githubusercontent.com/107245706/206854933-903af489-7458-4805-845c-5465177349da.png)

![sfrock](https://user-images.githubusercontent.com/107245706/206854939-cc0ed254-0a17-4205-89a7-e9a462bff66a.png)



`Farm ID: 82122`
`DC: mmaxxpeinn#1206`

**Reply by adamhannigan** [reply]
Nice work!

The rock has an 'ancient' feel to it 💪

**Comment by On1el** [comment]
**Observatory**
![FINALOBS](https://user-images.githubusercontent.com/104461938/207920101-83f23d0b-2745-4b79-b2b5-83fcc574cd3c.gif)
![OSOBS](https://user-images.githubusercontent.com/104461938/207920105-474522f8-95a6-450f-a919-f0055b5298dc.gif)
____________
**Sunflower Rock**
![СтатуяSFL5x4](https://user-images.githubusercontent.com/104461938/207915939-ab862016-349d-496d-b029-30237bb51162.png)
![СтатуяSFL5](https://user-images.githubusercontent.com/104461938/207915949-1e31e860-b7ee-4067-a7fe-8afd6d6a864a.png)
___________
**Carrot sword**
![MCG5](https://user-images.githubusercontent.com/104461938/207931626-c41ec5da-bcf2-42e9-9fe1-b9b3b4213d2b.gif)
![FinalMCGx4](https://user-images.githubusercontent.com/104461938/207931633-538d7a26-1894-409c-bfb2-d0469fb7fcee.gif)
![MC](https://user-images.githubusercontent.com/104461938/207658274-28734964-3f67-45bc-8c0d-03016101a959.png)
![MC2](https://user-images.githubusercontent.com/104461938/207658282-8e5321d8-80e8-4e52-b087-25e32ceec300.png)
![MCx4](https://user-images.githubusercontent.com/104461938/207658283-09f11de7-dc29-440e-85a5-eef4f4e7c522.png)
![MC2x4](https://user-images.githubusercontent.com/104461938/207658284-b909c443-e4e7-4491-b197-7c71d116eca5.png)
_________
**Sunflower Statue**
![HODL2x4](https://user-images.githubusercontent.com/104461938/207951434-2ecffdd1-c4be-4982-bf75-507c0fca36a2.png)
![HODL2](https://user-images.githubusercontent.com/104461938/207951436-d42972d0-1cbc-4353-a932-bc04e5dd7602.png)

![HODLx4](https://user-images.githubusercontent.com/104461938/207950073-fd72fa77-6f8a-47b9-b3bb-c5ad94d446d6.png)
![HODL](https://user-images.githubusercontent.com/104461938/207950074-53714c36-94dd-4589-a7e2-7f04570a0b76.png)

**Reply by adamhannigan** [reply]
Awesome job 👏 

With the observatory do you think it is possible to align them wall/roof/door closer to the other buildings in the game. The stone walls appear a little dark compared to the other stone walls in the game - also consider using the same doors as the other buildings 😄 

The rock is creative! I would suggest trying to include a yellow sunflower somewhere on the rock so it is still a "Sunflower Rock" 🏆

**Reply by adamhannigan** [reply]
@On1el Tower is definitely improved aesthetic wise 👌 

The animation timing seems a little too fast when the window is opening/closing

**Reply by adamhannigan** [reply]
<img width="141" alt="Screen Shot 2022-12-15 at 7 12 58 am" src="https://user-images.githubusercontent.com/11745561/207704369-6949fed2-e246-44a1-82e1-d64245b41d62.png">

Love this that it actually a polygon rock 👏 

Can we keep the Sunflower the same as it was before in the previous design?

**Comment by LittleEins** [comment]
### LittleEins Redesign
![redesign](https://user-images.githubusercontent.com/104184348/206885309-5711fc9e-f51f-4cae-9868-4ad3e354760a.png)
Minified Art:
![carrot-sword](https://user-images.githubusercontent.com/104184348/206840228-e0c868fe-9371-4120-9686-1a81b1894677.png)
![golden-cauli](https://user-images.githubusercontent.com/104184348/206885301-a36becf4-be53-4526-8cab-da4ed6fb19f2.png)

**Reply by adamhannigan** [reply]
Wow! That Golden cauliflower is incredible - amazing work 👏

**Comment by ghost** [comment]
**Nancy**

Preview:
![image](https://user-images.githubusercontent.com/114235416/206876520-3dcf22f3-092b-4820-8610-c379a9a98960.png)

Minifed:
![image](https://user-images.githubusercontent.com/114235416/206876495-5965f9fe-ec62-4d9a-b80c-943523b482d4.png)

---
**Sunflower Tombstone** 

Preview:
![image](https://user-images.githubusercontent.com/114235416/207181678-1319702d-6f97-40b3-825b-5aa9f82399db.png)

Minified:
![image](https://user-images.githubusercontent.com/114235416/207181898-446fc38e-6684-4d6f-bc2e-ec5a95d6d4e8.png)


Farm ID: 58821

**Reply by adamhannigan** [reply]
Looks much more aligned with other pixel assets now 😄

**Comment by esp123** [comment]
![scarecrow-reworkx3](https://user-images.githubusercontent.com/74337946/206884445-d8bbb9f2-8a57-4066-a80f-58d6a4592390.png)
![scarecrow-rework](https://user-images.githubusercontent.com/74337946/206884454-341ab16c-aecd-4ee0-a499-795a8fde0a01.png)

deeefault#0033

**Reply by adamhannigan** [reply]
This is a cute approach! The colours on these now feel closer to the current game's palette 👍

**Comment by sfwhat** [comment]
### Carrot sword 1x2 grid
![Sprite-0002](https://user-images.githubusercontent.com/95937808/206917924-d2d0e815-d2c4-4a44-a6eb-a0b17564894b.png)

up![Sprite-0004](https://user-images.githubusercontent.com/95937808/206917905-c5196b5e-9229-42d5-91e3-e6e283108c16.png)

down![Sprite-0003](https://user-images.githubusercontent.com/95937808/206917906-54ac372f-68ab-452f-88fe-f500ee69dce1.png)

---
Sfwhat
Farm 6127

**Reply by adamhannigan** [reply]
Nice work! Reminds me of Excalibur 😆

**Reply by sfwhat** [reply]
A new version
![sword2](https://user-images.githubusercontent.com/95937808/223479038-ff0b85a8-477d-43f1-9d0c-1d2211082f29.png)
Asset
![Carrotsword](https://user-images.githubusercontent.com/95937808/223479046-7a65cef9-4de4-44e4-93e4-73d940309c4d.png)
